Client/Frontend/AuthenticationManager/AppAuthenticator.swift

```diff
@@ -54,7 +54,7 @@ class AppAuthenticator {
         })
 
         let navController = UINavigationController(rootViewController: passcodeVC)
-        navController.modalPresentationStyle = .formSheet
+        navController.modalPresentationStyle = UIDevice.current.isPhone ? .fullScreen : .formSheet
```

Comment:
This whole line can be replaced by [UIModalPresentationStyle.automatic](https://developer.apple.com/documentation/uikit/uimodalpresentationstyle/automatic) I think

```suggestion
        navController.modalPresentationStyle = .automatic
```